eng/pipelines/runtime.yml

```diff
@@ -993,6 +993,48 @@ extends:
                     eq(variables['coreclrContainsChange'], true),
                     eq(variables['isRollingBuild'], true))
 
+      #
+      # Android arm64 devices and x64 emulators
+      # Build the whole product using CoreCLR and run functional tests
+      #
+      - template: /eng/pipelines/common/platform-matrix.yml
```

Comment:
Consider moving this to runtime-extra-platforms pipeline to reduce the load on devices and emulators.